src/atoms/RadioButton/__tests__/RadioButton.native.test.js

```diff
@@ -0,0 +1,196 @@
+          <Radio.Option value="3" title="Vue" size="large" testID="vueSelectId" />
+        </Radio>,
+      );
+      const component = getByTestId('reactSelectId');
+
+      act(() => {
+        fireEvent.pressIn(component);
+      });
+
+      expect(container).toMatchSnapshot();
+    });
+
+    test('snapshot testing when user presses out', () => {
+      const { container, getByTestId } = renderWithTheme(
+        <Radio>
+          <Radio.Option value="1" title="React" testID="reactSelectId" />
+          <Radio.Option value="2" title="Angular" testID="angularSelectId" />
+          <Radio.Option value="3" title="Vue" testID="vueSelectId" />
+        </Radio>,
+      );
+      const component = getByTestId('angularSelectId');
+
+      act(() => {
+        fireEvent.pressIn(component);
+        fireEvent.pressOut(component);
+      });
+      expect(container).toMatchSnapshot();
+    });
+
+    test('should render radio buttons of different sizes', () => {
+      const { container } = renderWithTheme(
+        <Radio>
+          <Radio.Option value="1" title="React" size="large" />
+          <Radio.Option value="2" title="Angular" size="medium" />
+          <Radio.Option value="3" title="Vue" size="large" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    test('should render radio buttons with helpTexts', () => {
+      const { container } = renderWithTheme(
+        <Radio>
+          <Radio.Option value="1" title="React" size="large" helpText="Click me!" />
+          <Radio.Option value="2" title="Angular" size="medium" helpText="Click me!" />
+          <Radio.Option value="3" title="Vue" size="small" />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    test('should render radio buttons with disabled state', () => {
+      const { container } = renderWithTheme(
+        <Radio>
+          <Radio.Option value="1" title="React" size="large" disabled helpText="Click me!" />
+          <Radio.Option value="2" title="Angular" size="medium" disabled helpText="Click me!" />
+          <Radio.Option value="3" title="Vue" size="small" disabled />
+        </Radio>,
+      );
+      expect(container).toMatchSnapshot();
+    });
+
+    test('should not render helpText when the size is small', () => {
+      const { queryByText } = renderWithTheme(
+        <Radio>
+          <Radio.Option value="1" title="React" size="small" disabled helpText="Fill details" />
+        </Radio>,
+      );
+      const helpTextComponent = queryByText('Fill details');
+      expect(helpTextComponent).toEqual(null);
+    });
+
+    test('should not render error when the size is small', () => {
+      const { queryByText } = renderWithTheme(
+        <Radio>
+          <Radio.Option
+            value="1"
+            title="React"
+            size="small"
+            disabled
+            errorText="Something is wrong!"
```

Comment:
Change title from `React` to maybe `Angular` or remove this `errorText` with `React` 😆